output_3.5.txt

```diff
@@ -0,0 +1,26 @@
+
+TASK: 3.5
```

Comment:
+ incorrect query - you should output `borough` instead of `name`
+ use `ISODate("2014-02-01")`  and `ISODate("2014-03-01")` instead of `ISODate("2014-02-11T00:00:00Z")` and ISODate("2014-03-11T00:00:00Z")
+ no need to use `"_id" : 1` as it is done automatically